src/app/state/wallet/types.ts

```diff
@@ -1,11 +1,14 @@
 import { BalanceDetails } from '../account/types'
 
 export enum WalletType {
-  Ledger = 'ledger',
+  UsbLedger = 'UsbLedger',
+  BleLedger = 'BleLedger',
```

Comment:
This requires state migration

Reply:
I suggest reverting and making migrations in a separate PR:
```suggestion
  UsbLedger = ledger',
  BleLedger = 'BleLedger',
```